### ISSUE TITLE
Reference posterior for pilot-pilot

### ISSUE THREAD

**Issue by MansMeg** [MansMeg/posteriordb#164]
I move @jarnefeltoliver comment here instead as an issue. To access the data and model just run:
```
remotes::install_github("MansMeg/posteriordb", subdir = "rpackage")
library(posteriordb)
po <- posterior("pilots-pilots")
sd <-  stan_data(po)
stan_code(po)
```
From Oliver:
Also it seems that pilots-pilots posterior in its current form is a bit problematic. With method_arguments = list(chains = 10, iter = 30000, warmup = 20000, thin = 10, seed = 4711, control = list(adapt_delta = 0.99, max_treedepth = 15))),
There are a couple of hundred divergent transitions after warmup and the max_treedepth gets exceeded. Maybe a (non-centered) reparametrization could be in order here? @paul-buerkner @avehtari

**Comment by avehtari** [comment]
I agree it's problematic and should be investigated. We want to have some difficult posteriors in db, too, but 1) it would be good to still have a good reference posterior draws, 2) if good reference posterior draws are not available, then the reference should be empty.

**Comment by MansMeg** [comment]
I suggest we leave it open as an issue for now and then I can see if it would be possible to solve this with a non-centered version, as we do with 8-schools. Then we could point this posterior to use the non-centered reference posterior.
